src/main/java/step3/game/RacingGame.java

```diff
@@ -1,37 +1,39 @@
 package step3.game;
 
 import step3.car.Car;
-import step3.car.RacingCar;
+import step3.strategy.MoveStrategy;
+
+import java.util.*;
 
-import java.util.ArrayList;
-import java.util.List;
 
 public abstract class RacingGame {
-    private List<Car> cars = new ArrayList<>();
+    private final List<Car> cars = new ArrayList<>();
+    private final Map<Integer, List<Integer>> gameHistory = new HashMap<>();
 
     public void addCar(Car car) {
         cars.add(car);
     }
 
-    public void deleteCar(Car car) {
-        cars.remove(car);
+    private void logGameHistory(int round){
+        List<Integer> list = new ArrayList<>();
+        cars.forEach(c-> list.add(c.currentProgress()));
```

Comment:
```suggestion
        cars.forEach(car -> list.add(car.currentProgress()));
```
람다 내부에서도 파라미터를 축약하지 않는것을 권장 드려요. 
https://velog.io/@iamjoo/clean-code-%EC%9E%91%EC%84%B1%EB%B2%95